Photon/IoTlib/src/pressure.cpp

```diff
@@ -0,0 +1,187 @@
+/******************************************************************
+Pressure sensor
```

Comment:
I notice that the BMP085 and it successor BMP180 have been discontinued by Adafruit. So I'm wondering if this can be generalized to include other pressure sensors that are still available? This is of course a more general problem that we'll run into with the existing "temperature" device, since it assumes use of the DHT11 or DHT22, and there are other sensors out there that could be used. I'm inclined to include this code pretty much as is for now, but would like to start thinking about how to make things easier to reuse for new devices.